memeid/src/test/scala/memeid/arbitrary/instances.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package memeid4s.arbitrary
+package memeid.arbitrary
```

Comment:
Shouldn't this be part of the `memeid4s`? Is Scala only.

Reply:
It needs to be here since other tests in the `memeid` package use it